packages/shopping/src/application.ts

```diff
@@ -99,4 +101,15 @@ export class ShoppingApplication extends BootMixin(
 
     this.bind(UserServiceBindings.USER_SERVICE).toClass(MyUserService);
   }
+
+  modifyApiSpecAndRebind() {
```

Comment:
Let's rename it as `updateOpenApiSpecWithSecurity`.

Reply:
This code will be obsolete in my next commit; based on our conversation yesterday.